packages/vertica-nodejs/test/integration/client/type-parser-override-tests.js

```diff
@@ -20,7 +19,7 @@ pool.connect(
   assert.success(function (client1, done1) {
     pool.connect(
       assert.success(function (client2, done2) {
-        var boolTypeOID = VerticaType.boolTypeOID
```

Comment:
Why did you remove the common type definitions?   Referring to values using the hard coded number is not as good of a practice than using a common definition.

Reply:
This wasn't working. It was either imported incorrectly or exported incorrectly in v-protocol, but boolTypeOID was being given a value of undefined, so the type parser for booleans wasn't being used. 

I agree that we need to refer to VerticaType when we can and I have already logged a jira for making these changes on a larger scale throughout the driver and driver tests.